package.json

```diff
@@ -53,18 +55,43 @@
     "broccoli-funnel": "^3.0.8",
     "ember-cli-babel": "^7.26.6",
     "ember-cli-htmlbars": "^5.7.1",
-    "ember-modifier": "^2.1.2 || ^3.0.0"
+    "ember-cli-typescript": "^4.2.1",
+    "ember-modifier": "^2.1.2 || ^3.0.0",
+    "tracked-built-ins": "^2.0.0"
   },
   "devDependencies": {
     "@ember/optional-features": "^2.0.0",
+    "@ember/render-modifiers": "^2.0.0",
```

Comment:
As mentioned above this would need to be in `dependencies`. I favour using a custom modifier anyway.